device/thunder_ripple_sdk/src/bootstrap/boot_thunder.rs

```diff
@@ -47,6 +47,7 @@ pub async fn boot_thunder(
     ext_client: ExtnClient,
     _plugin_param: ThunderPluginBootParam,
     device_manifest: &DeviceManifest,
+    local_dev: Option<bool>    
```

Comment:
Do we need to pass this as a separate param? How is it working in current version with extensions as shared library ?